tests/test_bmap_helpers.py

```diff
@@ -71,6 +71,7 @@ def test_get_file_system_type_symlink(self):
             os.symlink(fobj.name, lnk)
             fstype = BmapHelpers.get_file_system_type(lnk)
             self.assertTrue(fstype)
+            fobj.close()
```

Comment:
Can this be:

```
with tempfile.NamedTemporaryFile(...) as fobj:
   ...
```

Instead of manually closing? Otherwise I think it can fail if an exception is raised anywhere inside, including if self.assertTrue fails

Reply:
Yes, using a context manager works just the same with disorderfs and it's the better style. My initial patch didn't have it because I just copypasted the Debian patch which tried to be very minimal (and indentation changes can easily hide unintended changes). Thank you for making me aware of this! This is implemented now with my most recent push which also rebased this pull request on top of main. Thank you for so promptly merging the other stuff as well. :+1: